sklearn/metrics/tests/test_score_objects.py

```diff
@@ -526,3 +536,27 @@ def test_scoring_is_not_metric():
                          Ridge(), r2_score)
     assert_raises_regexp(ValueError, 'make_scorer', check_scoring,
                          KMeans(), cluster_module.adjusted_rand_score)
+
+
+def test_copy_classes():
+    # Test the various properties of copy_classes parameter added
+
+    X, y = make_blobs(random_state=0, centers=2)
+    clf = LogisticRegression(random_state=0)
+    clf.fit(X, y)
+
+    # copy_classes=None should not set labels
+    scorer = make_scorer(DummyScorerWithLabels, copy_classes=None)
+    assert_equal(scorer(clf, X, y), 0)
+
+    # by default, labels should be set
+    scorer = make_scorer(DummyScorerWithLabels)
+    assert_equal(scorer(clf, X, y), 1)
```

Comment:
This test is ok, but I'd like to also have a test with, say ``f1_score``, and using ``make_scorer`` with ``copy_classes=None`` and ``copy_classes='labels'`` and using the built-in ``f1_scorer`` object.

You can fit on more classes then you test on, similar to what you did in the other test.